### PR TITLE
E2E for config watcher

### PR DESCRIPTION
**Pull request by GunaKKIBM**

### CODE REVIEW

File: pkg/ibmcsidriver/config_watcher_test.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+		})
+	}
+}
+
+// FakeCreateCM ...
+func FakeCreateCM(kc k8sUtils.KubernetesClient) error {
+	data := make(map[string]string)
+	data["vpc_subnet_ids"] = "0757-229d3fcd-41da-42fc-a770-fddf5f415d9c,0767-d6884d9a-57ba-49a4-8914-4ac120b70a21"
+	cm := new(v1.ConfigMap)
+	cm.Data = data
```
Comment (GunaKKIBM) [inline pkg/ibmcsidriver/config_watcher_test.go:141]: Remove this

**Comment by contsto** [comment]
:thumbsup: Coverage increased from [%] to [80.8625%]

**Comment by contsto** [comment]
:thumbsup: Coverage increased from [%] to [80.5875%]

**Comment by contsto** [comment]
:thumbsup: Coverage increased from [%] to [81.9%]

**Pull request closed without merging**